### PR TITLE
Fix for PCRE2 checking and download

### PR DESCRIPTION
**Pull request by mdziczkowski**
Changes in PCRE2 related variables:

* PCRE2_VERSION -> numeric version only
* PCRE2_TARBALL -> "pcre2-${PCRE2_VERSION}.tar.gz"
* PCRE2_URL -> "https://github.com/PhilipHazel/pcre2/releases/download/pcre2-${PCRE2_VERSION}/${PCRE2_TARBALL}"

PCRE2_DIR="$(pwd)/pcre2-${PCRE2_VERSION}"

~~ mdziczkowski

### CODE REVIEW

File: configure.in [hunk 1]
```diff
@@ -414,9 +414,9 @@ else
  fi
 fi
 AC_MSG_NOTICE([Configuring PCRE2])
-PCRE2_VERSION="pcre2-10.39"
-PCRE2_TARBALL="${PCRE2_VERSION}.tar.gz"
-PCRE2_URL="https://github.com/PhilipHazel/pcre2/releases/download/${PCRE2_VERSION}/${PCRE2_TARBALL}"
+PCRE2_VERSION="10.39"
+PCRE2_TARBALL="pcre2-${PCRE2_VERSION}.tar.gz"
+PCRE2_URL="https://github.com/PhilipHazel/pcre2/releases/download/${PCRE2_TARBALL}"
```
Comment (tkrajcar) [inline configure.in:419]: This doesn't work. It needs to expand to `https://github.com/PhilipHazel/pcre2/releases/download/pcre2-10.39/pcre2-10.39.tar.gz`.

**Review by tkrajcar — CHANGES_REQUESTED** [review]

File: configure.in (hunk 1 above)
Comment (mdziczkowski) [reply configure.in:419]: I had noticed and wanted to fix it.

**Review by mdziczkowski — COMMENTED** [review]
See changes descriptions

**Comment by shawnw** [comment]
As with the other one, changes need to go in configure.in. Including some of this in an upcoming update.

**Comment by mdziczkowski** [comment]
Just a side note as I had looked at this again:

I had had then hardcoded a pcre2 version because I had not an idea how to cause the automatic detection of the latest avaliable version of it. This is a TODO

**Pull request closed without merging**